src/util/logger-utils.test.ts

```diff
@@ -0,0 +1,150 @@
+import { filterHttpRequests, SafeRawLogStream, UnsafeRawLogStream } from "./logger-utils";
+import { INFO, TRACE, DEBUG } from "bunyan";
+
+describe("Logger Utils", () => {
+
+	describe("filterHttpRequests", () => {
+
+		it("should return false for msg with http request keyword and not targeted logger", () => {
+			const record = {
+				msg: "GET - Im a filterable msg",
+				name: "NO-FILTEREDNAME"
+			};
+			expect(filterHttpRequests(record, "FILTEREDNAME")).toEqual(false);
```

Comment:
/nit
I understand `filterHttpRequests` is only exported for testing purpose. I suggest it could be better if the test is not against the implementation detail (`filterHttpRequests`), but instead testing on the `stdoutSpy.mock` not being called if filtered out.

Reply:
!!!!! I am having feelings here, I accept what you are saying, I would like to discuss this in a more more general standards kind of discussion though. I think we have a few areas that we could improve how we test and expose things etc.